src/instrumentsscene/view/instrumentsonscorelistmodel.cpp

```diff
@@ -153,17 +161,38 @@ void InstrumentsOnScoreListModel::loadOrders()
     TRACEFUNC;
 
     m_scoreOrders = repository()->orders();
```

Comment:
Please, add a check for emptiness here

Reply:
As a matter of fact, it should also test for a `CustomOrder` and add on if is missing.
Same change made in `src/instrumentsscene/view/instrumentspanelcontextmenumodel.cpp`.